src/webhooks/webhook.ts

```diff
@@ -42,6 +43,7 @@ const handleMergeRequest = async (payload: Webhook.Payload) => {
   // Link MR to record
   await linkMergeRequestToRecord(mr, record);
```

Comment:
This looks like an existing issue, but `record` might be `undefined` here. We should return if that is the case. Also, `linkMergeRequest` already calls `linkMergeRequestToRecord`.

```suggestion
  if (!record) {
    return null;
  }

  // Link MR to record
```